mypy.ini

```diff
@@ -1,2 +1,13 @@
 [mypy]
-ignore_missing_imports = True
+# Ensure consistent behavior across platforms
+python_version = 3.9
+platform = linux
+ignore_missing_imports = False
+strict = False
+warn_return_any = False
+warn_unused_configs = True
+disallow_untyped_defs = False
+exclude = node_modules
+# Force consistent stub resolution
+no_site_packages = False
+
```

Comment:
I started to add type checking (mypy) and linting (flake8) in this PR, but decided against it.  I'm going to leave the stuff related to lint in this PR, but not actually start enforcing its use until a later PR.  There are lint and type errors in python code outside of these changes.